src/QsCompiler/LanguageServer/LanguageServer.cs

```diff
@@ -123,9 +124,46 @@ public void Dispose()
         internal Task NotifyClientAsync(string method, object args) =>
             this.rpc.NotifyWithParameterObjectAsync(method, args);  // no need to wait for completion
 
+        internal Task<T> InvokeAsync<T>(string method, object args) =>
+           this.rpc.InvokeWithParameterObjectAsync<T>(method, args);
+
         internal Task PublishDiagnosticsAsync(PublishDiagnosticParams diagnostics) =>
             this.NotifyClientAsync(Methods.TextDocumentPublishDiagnosticsName, diagnostics);
 
+        internal async void CheckDotNetSdkVersion()
+        {
+            var installedVersionsOfDotNetSdk = DotNetSdkHelper.GetInstalledVersionsOfDotNetSdks();
+            if (installedVersionsOfDotNetSdk == null)
+            {
+                this.LogToWindow("Unable to detect .NET SDK versions", MessageType.Error);
+            }
+            else
+            {
+                if (!installedVersionsOfDotNetSdk.DotNetSdk31Installed
+                    && !installedVersionsOfDotNetSdk.DotNetSdk5Installed)
+                {
+                    const string dotnetSdkUrl = "https://dotnet.microsoft.com/download/dotnet-core/";
+                    this.LogToWindow($".NET Core SDK 3.1 or greater not found. Quantum Development Kit Extension requires .NET Core SDK version 3.1 or greater ({dotnetSdkUrl}).", MessageType.Error);
+                    var downloadAction = new MessageActionItem { Title = "Download" };
+                    var cancelAction = new MessageActionItem { Title = "No, thanks" };
+                    var selectedAction = await this.ShowDialogInWindowAsync(
+                        "Quantum Development Kit Extension requires either .NET Core SDK version 3.1 or greater. Please install the latest .NET Core SDK and restart Visual Studio.",
```

Comment:
Based on @rainersigwald's comment in #737, I believe we need to target the language server for `net5` in order to support both 3.1 and 5 SDKs. Since the language server targets `netcoreapp3.1`, we would still require specifically that .NET Core SDK 3.1 is installed, even with the Microsoft.Build.Locator update. This dialog message shown to users implies that having only .NET 5 SDK installed would work, though.

@vxfield Does this match your testing when using the language server on a computer that only has .NET 5 SDK installed - does it fail to load the project with this PR applied?